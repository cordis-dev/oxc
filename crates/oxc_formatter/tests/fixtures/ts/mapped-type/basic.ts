export type Bar<T> = {[P in keyof T]: string}
