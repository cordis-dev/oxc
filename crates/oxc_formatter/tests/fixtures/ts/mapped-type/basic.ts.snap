---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
export type Bar<T> = {[P in keyof T]: string}

==================== Output ====================
------------------------
{ bracketSpacing: true }
------------------------
export type Bar<T> = { [P in keyof T]: string };

-------------------------
{ bracketSpacing: false }
-------------------------
export type Bar<T> = {[P in keyof T]: string};

===================== End =====================
